build.gradle

```diff
@@ -91,6 +91,8 @@ dependencies {
 
 	compile("com.amazonaws:aws-java-sdk-kinesis:$awsSdkVersion", optional)
 
+	compile("com.amazonaws:amazon-kinesis-client:1.7.0", optional)
```

Comment:
I'll remove this dependency, when we are good with the solution.
I need it for stealing ideas :smile: